openforcefield/typing/engines/smirnoff/forcefield.py

```diff
@@ -828,9 +821,8 @@ def create_openmm_system(self,
         # Create an empty OpenMM System
         system = openmm.System()
 
-        # Set periodic boundary conditions if specified
-        if default_box_vectors is not None:
-            system.setDefaultPeriodicBoxVectors(default_box_vectors)
+        # Set periodic boundary conditions from topology
+        system.setDefaultPeriodicBoxVectors(topology.box_vectors)
```

Comment:
I'm not sure this works when `topology.box_vectors` is `None`.

Reply:
Good catch. It looks like we need to ensure that our setting of nonbonded methods takes into account that we don't have box vectors, and OpenMM will ignore them after that. Put the "if" statement back in 4f8d950